README.md

```diff
@@ -33,13 +33,10 @@ Try the autocomplete and the instantsearch results page on our [live demo](https
 
 We are happy to announce that version 3.x of our extension is now compatible with Magento 2.4. Review the [Customisation](https://github.com/algolia/algoliasearch-magento-2#customisation) section to learn more about the differences between our extension versions.
 
-The 1.x and 2.x versions of the extension are now End of Life (EOL). We no longer accept community PRs for those branches. 
-
 | Extension Version | End of Life |
-| --- | --- |
-| v1.x | Dec 2020 |
-| v2.x | Dec 2020 |
-| v3.x | N/A |
+|-------------------| --- |
+| v3.13.x           | N/A |
+| v3.14.x           | N/A |
```

Comment:
This also would be great to be more granular here and keep track of which Magento 2.4.x versions were compatible in the past (not easy to find in the release notes I guess) for each extension version.

Reply:
@mrahman3177 Let's move the Magento and PHP version here. Have we done any testing on older versions? I also think we should call out 3.7.x as being EOL as that was the last time we supported Autocomplete Hogan templates.